mlflow/pyfunc/backend.py

```diff
@@ -30,18 +30,23 @@ def __init__(self, config, workers=1, no_conda=False, install_mlflow=False, **kw
         self._nworkers = workers or 1
         self._no_conda = no_conda
         self._install_mlflow = install_mlflow
+        self._env_id = os.environ.get("MLFLOW_HOME", VERSION) if install_mlflow else None
 
     def prepare_env(self, model_uri, capture_output=False):
         local_path = _download_artifact_from_uri(model_uri)
         if self._no_conda or ENV not in self._config:
             return 0
         conda_env_path = os.path.join(local_path, self._config[ENV])
+
+        conda_env_name = get_or_create_conda_env(
+            conda_env_path, env_id=self._env_id, capture_output=capture_output
+        )
+
         command = 'python -c ""'
```

Comment:
Do we really need to run `python -c ""`?

Reply:
```
    if install_mlflow:
        if "MLFLOW_HOME" in os.environ:  # dev version
            install_mlflow = "pip install -e {} 1>&2".format(os.environ["MLFLOW_HOME"])
        else:
            install_mlflow = "pip install mlflow=={} 1>&2".format(VERSION)
```
this part is in `_execute_in_conda_env`, I think 'python -c ""' is for triggering it.